projects/kalidea/kaligraphi/src/lib/molecules/kal-molecules.module.ts

```diff
@@ -12,6 +13,7 @@ export * from './kal-rater/kal-rater.module';
 const exports = [
   KalTextareaModule,
   KalSelectModule,
+  KalAtomsModule,
```

Comment:
Why you should import AtomsModule in Molecules ?

Reply:
import only required module in kalSelectModule